### PR TITLE
Removing unnecessary ROCm installs.

### PR DESCRIPTION
**Pull request by Jorghi12**
Removing unnecessary installs since ROC 1.8.2 contains them.

### CODE REVIEW

File: docker/caffe2/jenkins/common/install_rocm.sh [hunk 1]
```diff
@@ -42,6 +36,7 @@ install_ubuntu() {
                    hipblas \
                    rocrand \
                    rocm-profiler \
+                   hcsparse \
```
Comment (bddppq) [inline docker/caffe2/jenkins/common/install_rocm.sh]: what about hcrng?

**Review by bddppq — COMMENTED** [review]
Let's fold it into #9633 ?

File: docker/caffe2/jenkins/common/install_rocm.sh (hunk 1 above)
Comment (Jorghi12) [reply docker/caffe2/jenkins/common/install_rocm.sh]: @bddppq Didn't see that PR. Yeah let's move all these changes there.

Comment (Jorghi12) [reply docker/caffe2/jenkins/common/install_rocm.sh]: Ah, I didn't see PR 9633.

**Pull request closed without merging**